### DISCUSSION TITLE
Is really needed to add HA Hacs integration and Frigate Card (Lovelace)?

### DISCUSSION THREAD

**Discussion by ryosoftware** [blakeblackshear/frigate#8570] [General]
Hello,
I use Frigate in a docker and, when I need to access my cameras I use the Frigate web UI.
I have installed HA also in a docker and I have installed the Frigate integration and also the Frigate Lovelace Card, but really I don't known if I need it or not.
I think I need the integration due to I use Alarmo software and I use person occupancy events to fire the alarm, but I think the Frigate Lovelace Card is useless to me? 
Is that True? I can uninstall or is mandatory to have it installed?
All of the  logs like this in the Frigate log are due to the Frigate Lovelace card or are due to the integrations?

`"GET /api/config HTTP/1.1" 200 23911 "XXXXXXXXXXXX" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36" "192.168.31.5"`

Thanks

**Comment by NickM-27** [comment]
there's no reason to have either if you don't use it, both are just ways to read from frigate. If you want to use automations or control frigate cameras then the integration is going to be the easiest (though it could all be done manually in mqtt / websocket).

frigate card is just a convenient way to add the cameras with live view, two way talk, etc. to the HA dashboard

**Reply by ryosoftware** [reply]
Thanks for the quick reply.
Frigate Lovelace Card serves for something else that show cameras live view at HA web UI?

![image](https://github.com/blakeblackshear/frigate/assets/31618496/29c07982-2a66-42a0-bece-59e05d7f47d0)

**Reply by NickM-27** [reply]
I mean the camera options in HA web UI are fairly lacking and frigate lovelace card was the first to support two way audio

**Reply by ryosoftware** [reply]
Thanks!
